pkg/devspace/pipeline/engine/pipelinehandler/commands/get_config_value.go

```diff
@@ -0,0 +1,56 @@
+package commands
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	devspacecontext "github.com/loft-sh/devspace/pkg/devspace/context"
+	"github.com/vmware-labs/yaml-jsonpath/pkg/yamlpath"
+	"gopkg.in/yaml.v3"
+	"mvdan.cc/sh/v3/interp"
+)
+
+func GetConfigValue(ctx *devspacecontext.Context, args []string) error {
+	ctx.Log.Debugf("get_config_value %s", strings.Join(args, " "))
+	if len(args) != 1 {
+		return fmt.Errorf("usage: get_config_value deployments.my-deployment.helm.chart.name")
+	}
+	rawConfig := ctx.Config.Raw()
+
+	nodePath, err := yamlpath.NewPath(getNodePath(args[0]))
+	if err != nil {
+		return err
+	}
+
+	out, err := yaml.Marshal(rawConfig)
+	if err != nil {
+		return err
+	}
+
+	var doc yaml.Node
+	err = yaml.Unmarshal(out, &doc)
+	if err != nil {
+		return err
+	}
+
+	nodes, err := nodePath.Find(&doc)
+	if err != nil {
+		return err
+	}
+
+	hc := interp.HandlerCtx(ctx.Context)
+	_, _ = hc.Stdout.Write([]byte(nodes[0].Value))
+	return nil
+}
+
+var endsWithBracketExp = regexp.MustCompile(`\[([^\]]+)\]$`)
+
+func getNodePath(path string) string {
+	if endsWithBracketExp.MatchString(path) {
+		return endsWithBracketExp.ReplaceAllString(path, "")
+	}
+
+	tokens := strings.Split(path, ".")
+	return strings.Join(tokens[0:], ".")
```

Comment:
why use `tokens[0:]` instead of `tokens`?

Reply:
ah used it as it is from patches but I guess we can use tokens instead of tokens[0:]